src/dynapi/SDL_dynapi.sym

```diff
@@ -1238,6 +1238,8 @@ SDL3_0.0.0 {
     SDL_RenderTexture9GridTiled;
     SDL_SetDefaultTextureScaleMode;
     SDL_GetDefaultTextureScaleMode;
+    SDL_SetWindowProgressState;
```

Comment:
These functions should not be sorted. Please revert the dynapi changes and run `gendynapi.py` instead.